.circleci/config.yml

```
@@ -366,6 +419,13 @@ workflows:
               only: /(.*)/
             tags:
               only: /^\d+\.\d+\.\d(.*)/ # npm tag + latest
+      - run-unity-tests:
+          <<: *all_branches_and_tags
+          requires:
+            - build-unity
+          filters:
+            branches:
+              only: /(master)|(release)|(staging)|(feat\/runtest)/
```

IMO, we should change this to run in every branch if possible.
```suggestion
              only: /(.*)/
```